test/parallel/test-buffer-alloc.js

```diff
@@ -1060,6 +1060,8 @@ assert.throws(function() {
   Buffer.allocUnsafe(0xFFFFFFFFF);
 }, RangeError);
 
+assert(Buffer.alloc.toString().length < 600, 'Buffer.alloc is not inlineable');
```

Comment:
Oh, and yes, in TurboFan this should be fixed now - it has limit only on AST complexity.